src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -130,6 +130,12 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
         public List<PSApplicationGatewayRequestRoutingRule> RequestRoutingRules { get; set; }
 
         [Parameter(
+             Mandatory = false,
+             ValueFromPipelineByPropertyName = true,
+             HelpMessage = "The list of redirect configuration")]
+        public List<PSApplicationGatewayRedirectConfiguration> RedirectConfigurations { get; set; }
```

Comment:
@akshaysngupta even though this parameter is a list, it should still have a singular noun. That is the guidance from the [Strongly Encouraged Developer Guidelines](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx) from PowerShell.

_Avoid using plural names for parameters whose value is a single element. This includes parameters that take arrays or lists because the user might supply an array or list with only one element._

_Plural parameter names should be used only in those cases where the value of the parameter is always a multiple-element value. In these cases, the cmdlet should verify that multiple elements are supplied, and the cmdlet should display a warning to the user if multiple elements are not supplied._